fedora-toolbox

```diff
@@ -246,6 +250,47 @@ enter()
 )
 
 
+list()
+(
+    if [ $1 = 1 ]; then
+        echo -e "${LBC}Images created by fedora-toolbox${NC}"
+        output=$($prefix_sudo podman images \
+                --filter "label=com.redhat.component=fedora-toolbox" \
+                --format "table {{.ID}} {{.Repository}} {{.Tag}} {{.Created}}")
+        if [[ $output = "" ]]; then
+            echo "No images available"
+        else
+            echo "$output"
+        fi
+    fi
+    if [ $2 = 1 ]; then
+        echo -e "${LBC}Containers created by fedora-toolbox (currently running containers in green)${NC}"
+        output=$($prefix_sudo podman ps \
+                --all \
+                --filter "label=com.redhat.component=fedora-toolbox" \
+                --format "table {{.ID}} {{.Names}} {{.Image}} {{.Created}} {{.Status}}")
```

Comment:
Which version of `podman` were you developing against?

I tried `podman-0.12.1.2-1.git9551f6b` and `podman-1.0.0-1.git82e8011`, and `podman ps` is not handling `table` at all, unlike `podman images`.

For example:
```
$ podman ps --all --filter "label=com.redhat.component=fedora-toolbox" --format "table {{.ID}} {{.Names}} {{.Image}} {{.Created}} {{.Status}}"
table 21f509790423 fedora-toolbox-rishi:29 localhost/fedora-toolbox-rishi:29 46 hours ago Created
table c86c1d455331 fedora-toolbox-rishi:28 localhost/fedora-toolbox-rishi:28 9 days ago Created

$ podman images --filter "label=com.redhat.component=fedora-toolbox" --format "table {{.ID}} {{.Repository}} {{.Tag}} {{.Created}}"
IMAGE ID       REPOSITORY                                      TAG   CREATED
ec61775deac1   localhost/fedora-toolbox-rishi                  29    46 hours ago
45b540de989b   localhost/fedora-toolbox-rishi                  28    9 days ago
...
```

Reply:
I was using podman version 0.10.1, as I did not update my Silverblue in VM, my bad. However, when using the --format flag, podman ps omits table headings and I don't see any other flag that would print them anyway. For now, we might need to print out the output with the default columns.

Reply:
Ok. I see that you already filed https://github.com/containers/libpod/issues/2221 for the `podman` regression.